frontend/src/components/molecules/ExpandableSection/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useRef, useState } from 'react';
+import './style.css';
+import { ArrowDownIcon, ArrowUpIcon } from '../../atoms/icons/fr-fi-icons';
+
+type Props = {
+  title: string;
+};
+
+export const ExpandableSection: React.FC<Props> = ({ title, children }) => {
+  const [expanded, setExpanded] = useState(false);
+  const ref = useRef(null);
```

Comment:
Shouldn't we rename `const ref` with another less generic name `const expandableRef` ?